xml/System.Data.Common/DbConnection.xml

```diff
@@ -428,9 +428,18 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a value that indicates whether this <see cref="T:System.Data.Common.DbConnection" /> instance supports the <see cref="T:System.Data.Common.DbBatch" /> class.</summary>
+        <value>
+          <see langword="true" /> if this instance supports the <see cref="T:System.Data.Common.DbBatch" /> class; otherwise, <see langword="false" />.
```

Comment:
```suggestion
          <see langword="true" /> if this instance supports the <see cref="T:System.Data.Common.DbBatch" /> class; otherwise, <see langword="false" />. The default is <see langword="false" />.
```